src/Pages/OATEditorPage/Internal/OATErrorHandlingWrapper.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { PrimaryButton, Modal } from '@fluentui/react';
+import { useTranslation } from 'react-i18next';
+import { SET_OAT_ERROR } from '../../../Models/Constants/ActionTypes';
+import { getEditorPageStyles } from '../OATEditorPage.Styles';
+import { IOATEditorState } from '../OATEditorPage.types';
+import { IAction } from '../../../Models/Constants/Interfaces';
+
+interface IOATErrorHandlingWrapper {
+    dispatch?: React.Dispatch<React.SetStateAction<IAction>>;
+    state?: IOATEditorState;
```

Comment:
Doesn't the `?` mean optional?

Both state & dispatch appear to be required for this component to work correctly?

Re: interface name, what's the pattern used elsewhere in the code base? Should it end with `Props`? I.e. `IOATErrorHandlingWrapperProps`?